docs/rl-algorithms/rpo.md

```diff
@@ -0,0 +1,344 @@
+        | Hopper-v2                 | 2387.39 ± 645.41                                             | 2272.78 ± 588.66                                         |
+        | InvertedDoublePendulum-v2 | 5630.91 ± 377.93                                             | 5661.29 ± 316.04                                         |
+        | Reacher-v2                | -4.61 ± 0.53                                                 | -4.24 ± 0.25                                             |
+        | Swimmer-v2                | 132.07 ± 9.92                                                | 141.37 ± 8.70                                            |
+        | Pusher-v2                 | -33.93 ± 8.55                                                | -26.22 ± 2.52                                            |
+                
+        
+        Learning curves:
+        ![](../rpo/mujoco_v2_part2.png)
+
+        Tracked experiments:     
+
+        <iframe loading="lazy" src="https://wandb.ai/openrlbenchmark/cleanrl/reports/RPO-on-Mujoco_v2-Part-2--VmlldzozMjU4MzI1" style="width:100%; height:500px" title="RPO-mujoco-v2-part1"></iframe>
+
+        Results with `rpo_alpha=0.5` (not tuned) on the tuned environments:
+
+        |                           | ppo_continuous_action_8M ({'tag': ['v1.0.0-13-gcbd83f6']})   | rpo_continuous_action ({'tag': ['pr-331']})   |
+        |:--------------------------|:-------------------------------------------------------------|:----------------------------------------------|
+        | Ant-v2                    | 2495.65 ± 991.65                                             | -7.81 ± 3.57                                  |
+        | HalfCheetah-v2            | 2722.03 ± 1231.28                                            | 2605.06 ± 1183.30                             |
+        | Hopper-v2                 | 2356.83 ± 650.91                                             | 1609.79 ± 164.16                              |
+        | InvertedDoublePendulum-v2 | 5675.31 ± 244.34                                             | 274.78 ± 16.40                                |
+        | Reacher-v2                | -4.67 ± 0.48                                                 | -66.55 ± 0.20                                 |
+        | Swimmer-v2                | 131.53 ± 9.94                                                | 114.34 ± 3.95                                 |
+        | Pusher-v2                 | -33.46 ± 8.41                                                | -275.09 ± 15.65                               |
+
+        Learning curves:
+        ![](../rpo/mujoco_v2_part2_0_5.png)
+
+        <!-- Tracked experiments: <iframe loading="lazy" src="https://wandb.ai/openrlbenchmark/cleanrl/reports/RPO-alpha-0-5-on-Mujoco_v2-Part-2--VmlldzozMjU4MzQ0" style="border:none;height:1024px;width:100%"> -->
```

Comment:
Is this duplicate info since we have the "Failure case of `rpo_alpha=0.5`:" down below?

Reply:
Oh nvm I see what you did there